source/analysis/AM3DBinningAB.cpp

```diff
@@ -472,8 +472,8 @@ void AM3DBinningAB::setSumRangeMin(int sumRangeMin)
     cacheUpdateRequired_ = true;
     dirtyIndices_.clear();
     reviewState();
-    emitValuesChanged();
```

Comment:
Hmm, forgot to finish that thought.  I have found that analysis blocks behave better when all the necessary pieces of the function have been dealt with first, then the notifications are sent out once they are all done.